cpp/include/raft/sparse/neighbors/knn.cuh

```diff
@@ -59,7 +62,7 @@ namespace raft::sparse::neighbors {
  * @param[in] metric distance metric/measure to use
  * @param[in] metricArg potential argument for metric (currently unused)
  */
-template <typename value_idx = int, typename value_t = float, int TPB_X = 32>
```

Comment:
You'll need to go through cuml and make sure this isn't going to break anything once we merge it.

Reply:
removing that code from this PR because this logic now lives in cuvs.